tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php

```diff
@@ -1475,6 +1475,41 @@ public function testSearch()
         $this->assertCount(1, $crawler->filter($this->entryDataTestAttribute));
     }
 
+    /**
+     * @group plop
+     */
+    public function testActionInSearchResults()
+    {
+        $this->logInAs('admin');
+        $client = $this->getTestClient();
+
+        $config = $this->getLoggedInUser()->getConfig();
+        $config->setActionMarkAsRead(ConfigEntity::REDIRECT_TO_CURRENT_PAGE);
```

Comment:
ah this setting is used for delete action also?

Reply:
yes